### ISSUE TITLE
Hugo CSS minify breaks AMP validation

### ISSUE THREAD

**Issue by braderhart** [gohugoio/hugo#6699]
## Description
Hugo is stripping the space after a parenthesis when doing minification. This happens when doing both resource minification, or when using the `--minify` option.

Un-minified:
```css
body {
	-webkit-animation: -amp-start 8s steps(1,end) 0s 1 normal both;
	-moz-animation: -amp-start 8s steps(1,end) 0s 1 normal both;
	-ms-animation: -amp-start 8s steps(1,end) 0s 1 normal both;
	animation: -amp-start 8s steps(1,end) 0s 1 normal both
}
```

Minified:
```css
body{-webkit-animation:-amp-start 8s steps(1,end)0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end)0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end)0s 1 normal both;animation:-amp-start 8s steps(1,end)0s 1 normal both}
```

The issue is here `steps(1,end)0s`.

Oddly, when testing with the minify module Hugo is using [here](https://go.tacodewolff.nl/minify) it works correctly:

```css
body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}
```

## What version of Hugo are you using?
```Hugo Static Site Generator v0.62.0/extended linux/amd64 BuildDate: unknown```

## Does the issue happen in the latest release?
Yes

**Comment by braderhart** [comment]
Duplicate of #https://github.com/gohugoio/hugo/issues/6563

**Comment by apoorvmote** [comment]
Same issue with newly added built in minify options. Hugo version 0.68.3

`The mandatory text inside tag 'head > style[amp-boilerplate]' is missing or incorrect.`

**Comment by pperzyna** [comment]
I created an issue on tdewolff [#300](https://github.com/tdewolff/minify/issues/300), because it is an intendant behavior of that lib.

**Comment by tdewolff** [comment]
This is fixed in v2.6.2 of `minify`. Hugo should probably update its version of `minify` to fix this problem.
